settings.gradle.kts

```diff
@@ -1,11 +1,19 @@
-rootProject.name = "kover-gradle-plugin"
+rootProject.name = "kover"
 
 pluginManagement {
-    val kotlinVersion: String by settings
+    includeBuild("build-logic")
+
     plugins {
-        kotlin("jvm") version kotlinVersion
+        kotlin("jvm") version embeddedKotlinVersion
+    }
+}
+
+dependencyResolutionManagement {
+    versionCatalogs {
+        create("libs")
     }
 }
 
-include(":toolset:kover-cli")
-include(":toolset:kover-offline-runtime")
\ No newline at end of file
+include(":kover-gradle-plugin")
```

Comment:
Also, I haven't noticed any changes in publication-related functionality. Are you sure root project won't be published accidentally?

Reply:
Yes, the root project cannot be published, because the `maven-publish` plugin is not applied in it and, accordingly, there are no publishing tasks.